plugins/modules/aws_eks_fargate_profile.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/python
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: aws_eks_fargate_profile
+version_added: 1.0.0
```

Comment:
```suggestion
version_added: 3.2.0
```

if everything goes well